kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,40 +1,43 @@
-"""Project settings."""
+"""Project settings. There is no need to edit this file unless you want to change values
+from the Kedro defaults. For further information, including these default values, see
+https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
-# Instantiate and list your custom project hooks here
+#############
+# Lifecycle #
+#############
+
+# Instantiated project hooks.
 # from {{cookiecutter.python_package}}.hooks import ProjectHooks
 # HOOKS = (ProjectHooks(),)
 
-# List the installed plugins for which to disable auto-registry
+# Installed plugins for which to disable hook auto-registration.
 # DISABLE_HOOKS_FOR_PLUGINS = ("kedro-viz",)
 
-# Define where to store data from a KedroSession. Defaults to BaseSessionStore.
+# Class that manages storing KedroSession data.
 # from kedro.framework.session.store import ShelveStore
 # SESSION_STORE_CLASS = ShelveStore
-
-# Define keyword arguments to be passed to `SESSION_STORE_CLASS` constructor
+# Keyword arguments to pass to the `SESSION_STORE_CLASS` constructor.
 # SESSION_STORE_ARGS = {
 #     "path": "./sessions"
 # }
 
-# Define custom context class. Defaults to `KedroContext`
+# Class that manages Kedro's library components.
 # CONTEXT_CLASS = KedroContext
```

Comment:
This is the only one that doesn't have an import above it so maybe it'd be a good idea to include that as well.